### DISCUSSION TITLE
WinCommandAgent: transmitting bypass route to_ Server poll timout

### DISCUSSION THREAD

**Discussion by ber-serker** [angristan/openvpn-install#1100] [Help]
### Make sure your check these beforehand!

- [X] Issues - https://github.com/angristan/openvpn-install/issues
- [X] README and FAQ - https://github.com/angristan/openvpn-install
- [X] Wiki - https://github.com/angristan/openvpn-install/wiki
- [X] Discussions - https://github.com/angristan/openvpn-install/discussions

### Server OS

Ubuntu 20

### OpenVPN version

Latest

### Client

Latest
### What is the bug?

Ubuntu server, Windows client and angristan openvpn script. So I've been using this configuration since 2019 and I had no problems. Recently I can't connect to my server. And I tried several encryptions, DNS servers, and ..... Didn't work out. 

### Relevant log output

```shell
⏎9/27/2022, 8:30:04 PM EVENT: WAIT ⏎9/27/2022, 8:30:04 PM Transport Error: socket_protect error (UDP)
⏎9/27/2022, 8:30:04 PM Client terminated, restarting in 2000 ms...
⏎9/30/2022, 6:28:40 PM EVENT: DISCONNECTED ⏎9/30/2022, 6:29:48 PM OpenVPN core 3.git::662eae9a win x86_64 64-bit built on Oct 27 2020 12:49:07
⏎9/30/2022, 6:29:48 PM Frame=512/2048/512 mssfix-ctrl=1250
⏎9/30/2022, 6:29:48 PM UNUSED OPTIONS
2 [explicit-exit-notify] 
5 [resolv-retry] [infinite] 
6 [nobind] 
7 [persist-key] 
8 [persist-tun] 
10 [verify-x509-name] [server_XnZPdddfjdhSAsgL] [name] 
12 [auth-nocache] 
14 [tls-client] 
16 [tls-cipher] [TLS-ECDHE-ECDSA-WITH-AES-128-GCM-SHA256] 
17 [ignore-unknown-option] [block-outside-dns] 
18 [block-outside-dns] 
19 [verb] [3] 
⏎9/30/2022, 6:29:48 PM Contacting **************:1194 via UDP
⏎9/30/2022, 6:29:48 PM WinCommandAgent: transmitting bypass route to ************
{
	"host" : "********",
	"ipv6" : false
}

⏎9/30/2022, 6:29:48 PM EVENT: RESOLVE ⏎9/30/2022, 6:29:48 PM EVENT: WAIT ⏎9/30/2022, 6:29:48 PM Connecting to [**********]:1194 (*************) via UDPv4
⏎9/30/2022, 6:29:58 PM Server poll timeout, trying next remote entry...
⏎9/30/2022, 6:29:58 PM EVENT: RECONNECTING ⏎9/30/2022, 6:29:58 PM EVENT: RESOLVE ⏎9/30/2022, 6:29:58 PM EVENT: WAIT ⏎9/30/2022, 6:29:58 PM Contacting ***********:1194 via UDP
⏎9/30/2022, 6:29:58 PM WinCommandAgent: transmitting bypass route to ***************
{
	"host" : "*************",
	"ipv6" : false
}

⏎9/30/2022, 6:29:58 PM Connecting to [45.14.227.44]:1194 (45.14.227.44) via UDPv4
⏎9/30/2022, 6:30:08 PM Server poll timeout, trying next remote entry...
⏎9/30/2022, 6:30:08 PM EVENT: RECONNECTING ⏎9/30/2022, 6:30:08 PM EVENT: RESOLVE ⏎9/30/2022, 6:30:08 PM EVENT: WAIT ⏎9/30/2022, 6:30:08 PM Contacting *************:1194 via UDP
⏎9/30/2022, 6:30:08 PM WinCommandAgent: transmitting bypass route to 45.14.227.44
{
	"host" : "*************",
	"ipv6" : false
}

⏎9/30/2022, 6:30:08 PM Connecting to *****************]:1194 (**********) via UDPv4
⏎
```

**Comment by aes256ctr** [comment]
I got the same problem.
I create a latest open source openvpn server on centos7.9.
I can connect to it using the corresponding open souce windows openvpn 2.6 client.
But I can't use the latest openvpn connect(commercial version).

WHY?
I increated the `Verb` to 10, but I still got nothing useful.

**Comment by isla-blanca** [comment]
hi all, i have the same problem

My config work on a synology NAS with version 7.2.2_72806 and until 3 days ago, work perfectly.
Client 3.5 on windows 11 

my log:

[Dec 20, 2024, 10:56:27] OpenVPN core 3.10.1 win x86_64 64-bit OVPN-DCO built on Oct 14 2024 13:53:17
⏎[Dec 20, 2024, 10:56:27] Frame=512/2112/512 mssfix-ctrl=1250
⏎[Dec 20, 2024, 10:56:27] EVENT: RESOLVE ⏎[Dec 20, 2024, 10:56:27] Contacting 88.4x.xx.xx:1194 via UDP
⏎[Dec 20, 2024, 10:56:27] EVENT: WAIT ⏎[Dec 20, 2024, 10:56:27] WinCommandAgent: transmitting bypass route to 88.4x.xx.xx
{
	"host" : "88.4x.xx.xx",
	"ipv6" : false
}

⏎[Dec 20, 2024, 10:56:28] Connecting to [88.4x.xx.xx]:1194 (88.4x.xx.xx) via UDP
⏎[Dec 20, 2024, 10:56:37] Server poll timeout, trying next remote entry...
⏎[Dec 20, 2024, 10:56:37] EVENT: RECONNECTING ⏎[Dec 20, 2024, 10:56:37] EVENT: RESOLVE ⏎[Dec 20, 2024, 10:56:37] Contacting 88.4x.xx.xx:1194 via UDP
⏎[Dec 20, 2024, 10:56:37] EVENT: WAIT ⏎[Dec 20, 2024, 10:56:37] WinCommandAgent: transmitting bypass route to 88.4x.xx.xx
{
	"host" : "88.4x.xx.xx",
	"ipv6" : false
}

⏎[Dec 20, 2024, 10:56:38] Connecting to [88.4x.xx.xx]:1194 (88.4x.xx.xx) via UDP
⏎[Dec 20, 2024, 10:56:43] EVENT: DISCONNECTED ⏎[Dec 20, 2024, 11:10:35] EVENT: DISCONNECTED ⏎[Dec 20, 2024, 11:10:42] OpenVPN core 3.10.1 win x86_64 64-bit OVPN-DCO built on Oct 14 2024 13:53:17
⏎[Dec 20, 2024, 11:10:42] Frame=512/2112/512 mssfix-ctrl=1250
⏎[Dec 20, 2024, 11:10:42] EVENT: RESOLVE ⏎[Dec 20, 2024, 11:10:42] Contacting 88.4x.xx.xx:1194 via UDP
⏎[Dec 20, 2024, 11:10:42] EVENT: WAIT ⏎[Dec 20, 2024, 11:10:42] WinCommandAgent: transmitting bypass route to 88.4x.xx.xx
{
	"host" : "88.4x.xx.xx",
	"ipv6" : false
}

⏎[Dec 20, 2024, 11:10:42] Connecting to [88.4x.xx.xx]:1194 (88.4x.xx.xx) via UDP
⏎[Dec 20, 2024, 11:10:52] Server poll timeout, trying next remote entry...
⏎[Dec 20, 2024, 11:10:52] EVENT: RECONNECTING ⏎[Dec 20, 2024, 11:10:52] EVENT: RESOLVE ⏎[Dec 20, 2024, 11:10:52] Contacting 88.4x.xx.xx:1194 via UDP
⏎[Dec 20, 2024, 11:10:52] EVENT: WAIT ⏎[Dec 20, 2024, 11:10:52] WinCommandAgent: transmitting bypass route to 88.4x.xx.xx
{
	"host" : "88.4x.xx.xx",
	"ipv6" : false
}


there is someone with a solution?

regards
